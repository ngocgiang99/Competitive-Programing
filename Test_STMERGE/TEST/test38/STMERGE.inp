6
7 8
243398682 830595973 57027335 97802620 389165756 765030000 341293860 726664093 
164638828 657518019 261394824 811205544 169014958 86659732 965644248 385951955 
739289927 602636855 843427835 581717651 86238268 315752001 226689556 610921117 
608285650 665953928 763257992 83847910 56711822 314104608 277124094 812972451 
577219382 304460894 44473177 575151432 361132749 513523416 750827966 967203468 
93126810 577034595 843509395 182329442 921804321 922087186 70443389 119684096 
701262119 778513213 396413025 116919004 342114737 339800950 599325539 826217641 
10 8
298181353 76174173 816969382 865330603 127655831 480107308 954968667 223851025 
514368595 180204173 201754663 900912986 505349186 258161716 68097873 903540315 
487840456 430906133 729285469 103090730 729571723 197978727 910390800 188017691 
380390000 561499549 376785264 561349060 825082267 665303661 765361600 611428089 
588139378 577194188 808856967 88631759 8873113 160312047 505950600 70231708 
672363510 273199435 443939530 918050804 322712238 500446869 356088465 989032109 
463692326 22529473 401384034 813217755 723879938 213738502 556221955 418003667 
449460776 665933015 520538999 388185387 716391100 741191147 329111813 490095287 
255587876 297840437 646719792 901216653 413959941 63060203 332701887 409359361 
249747011 839336322 98532534 727023784 633073924 180815279 476539482 293878360 
2 7
766766385 768593445 771124450 4217054 235318014 451791716 289866633 
471014027 155387848 491652523 727291078 200588577 321801682 67752654 
3 1
35426211 
752444131 
474758873 
2 9
179342910 237684024 544830258 407691755 584990068 272747842 369119106 176545964 204672859 
820269660 17940229 993947080 33942792 216940074 428226800 361797378 364128718 75216530 
5 5
139258180 881384377 495491601 210194473 464325950 
970255363 125839601 462836447 472215506 378608150 
80402318 373286216 682221396 876915927 58452571 
989076585 675067994 338979690 755226638 867210120 
305680312 520208693 479696796 563156520 430680707 
