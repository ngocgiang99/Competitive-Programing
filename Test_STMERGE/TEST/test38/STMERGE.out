173946339
174706707
204805631
35426211
197283139
305680312
